packages/main/cypress/specs/DateRangePicker.cy.tsx

```diff
@@ -0,0 +1,575 @@
+			.as("dateRangePicker")
+			.shadow()
+			.find("ui5-input")
+			.as("input")
+			.realClick()
+			.realPress("End")
+			.realPress(["Control", "Shift", "PageUp"]);
+
+		cy.get("@dateRangePicker")
+			.should("have.attr", "value", "Jul 16, 2020 @ Jul 29, 2021");
+
+		cy.get("@input")
+			.realPress(["Control", "Shift", "PageDown"]);
+
+		cy.get("@dateRangePicker")
+			.should("have.attr", "value", "Jul 16, 2020 @ Jul 29, 2020");
+
+		cy.get("@input")
+			.realPress("Home")
+			.realPress(["Control", "Shift", "PageDown"]);
+
+		cy.get("@dateRangePicker")
+			.should("have.attr", "value", "Jul 16, 2019 @ Jul 29, 2020");
+
+		cy.get("@input")
+			.realPress(["Control", "Shift", "PageUp"]);
+
+		cy.get("@dateRangePicker")
+			.should("have.attr", "value", "Jul 16, 2020 @ Jul 29, 2020");
+	});
+
+	it("Enter keyboard key confirms the date range in the input field", () => {
+		cy.mount(<DateRangePicker formatPattern="MMM d, y" delimiter="@"></DateRangePicker>);
+
+		cy.get<DateRangePicker>("[ui5-daterange-picker]")
+			.as("dateRangePicker")
+			.shadow()
+			.find("ui5-input")
+			.as("input")
+			.realClick()
+			.realType("Jul 17, 2020 @ Jul 16, 2020")
+			.realPress("Enter");
+
+		cy.get("@dateRangePicker")
+			.should("have.attr", "value", "Jul 16, 2020 @ Jul 17, 2020");
+	});
+
+	it("Focus out of the input field confirms the date range", () => {
+		cy.mount(<DateRangePicker formatPattern="MMM d, y" delimiter="@"></DateRangePicker>);
+
+		cy.get<DateRangePicker>("[ui5-daterange-picker]")
+			.as("dateRangePicker")
+			.shadow()
+			.find("ui5-input")
+			.as("input")
+			.realClick()
+			.realType("Jul 17, 2020 @ Jul 16, 2020")
+			.realPress("Tab");
+
+		cy.get("@dateRangePicker")
+			.should("have.attr", "value", "Jul 16, 2020 @ Jul 17, 2020");
+	});
+
+	it("Delimiter is part of the format pattern", () => {
+		cy.mount(<DateRangePicker formatPattern="yyyy-MM-dd"></DateRangePicker>);
+
+		cy.get<DateRangePicker>("[ui5-daterange-picker]")
+			.as("dateRangePicker")
+			.shadow()
+			.find("ui5-input")
+			.as("input")
+			.realClick()
+			.realType("2020-09-09 - 2020-10-10")
+			.realPress("Enter");
+
+		cy.get("@input")
+			.should("have.attr", "value-state", "None");
+	});
+
+	it("Month on next button press", () => {
+		cy.mount(<DateRangePicker value="09/09/2020 - 10/10/2020" formatPattern="dd/MM/yyyy"></DateRangePicker>);
+
+		cy.get<DateRangePicker>("[ui5-daterange-picker]")
+			.as("dateRangePicker")
+			.shadow()
+			.find("ui5-calendar")
+			.as("calendar")
+			.shadow()
+			.find(".ui5-calheader")
+			.as("calendarHeader");
+
+		cy.get("@dateRangePicker")
+			.shadow()
+			.find("ui5-input")
+			.realClick()
+			.realPress("F4");
+
+		cy.get("@calendarHeader")
+			.find("[data-ui5-cal-header-btn-next]")
+			.realClick();
+
+		cy.get("@calendarHeader")
+			.find("[data-ui5-cal-header-btn-month]")
+			.as("monthButton")
+			.should("have.text", "October");
+	});
+
+	it("startDateValue and endDateValue getters when single value", () => {
```

Comment:
This as well, for example - "Check the startDateValue and endDateValue, when single value is provided"